From defcd2a8ff66b7b2d8fbaa7fc0f1ed7d6aae7bf8 Mon Sep 17 00:00:00 2001
From: Duy Bao Dang <duy.dang.yb@renesas.com>
Date: Fri, 8 May 2020 08:45:52 +0700
Subject: [PATCH 540/557] ARM: dts: r8a77470: Change power-domain for WDT.

This patch uses "SYSC always on" power-domain for WDT because
"cpg_clocks" domain causes bug in reboot function.

Signed-off-by: Duy Bao Dang <duy.dang.yb@renesas.com>
---
 arch/arm/boot/dts/r8a77470.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index da925fa..2758b86 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -97,7 +97,7 @@
 				     "renesas,rcar-gen2-wdt";
 			reg = <0 0xe6020000 0 0x0c>;
 			clocks = <&mstp4_clks R8A77470_CLK_RWDT>;
-			power-domains = <&cpg_clocks>;
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			status = "disabled";
 		};
 
-- 
2.7.4

